benchmarks/models.py

```diff
@@ -0,0 +1,77 @@
+import math
+
+import torch
+import torch.nn as nn
+
+
+class EmbeddingLayer(nn.Embedding):
```

Comment:
some docstring to document why do we wrap those underlying classes here will be useful.

Reply:
I think it is added to make initialization of the weights easier. I think you can just add that in the docstring here.

Reply:
Done, added the comment.

Is there a reason we prefer normalized weights in a given range as opposed to the default weight initialization? Should we be recommending users to follow this?

Reply:
I don't know. I think it is unlike to matter much in this benchmarking test.

Reply:
Added a comment to investigate more. If we don't need the weight initialization then we can use the layer directly.